styles/project-viewer.less

```diff
@@ -137,6 +137,10 @@ project-viewer {
 	span.icon {
 		font-family: @font-family;
 	}
+
+	span.icon::before {
+		font-size: 16px;
```

Comment:
Please consider using the `@component-icon-size` instead of the hard coded `16px`.